src/components/stack/index.jsx

```diff
@@ -70,7 +70,7 @@ Stack.propTypes = {
 	inline: PropTypes.bool,
 	/** The justification of the elements within the component.
 	    Relates to CSS Flex's `justify-content` */
-	justification: PropTypes.oneOf(["start", "center", "end"]),
+	justification: PropTypes.oneOf(["start", "center", "end", "space-between"]),
```

Comment:
You'll need to add a corresponding rule to the `_index.scss` as well. Something like:
```
.c-stack[data-style-justification="space-between"] {
	justify-content: space-between;
}
```
You'll see the other justification rules in that file and you can follow the same pattern.